src/inference.py

```diff
@@ -74,26 +74,29 @@ def predict(model: torch.nn.Module, data_loader: Generator[torch.tensor, None, N
         return (predicted_probability > threshold).reshape((-1, 1))
 
 
-def load_model(path):
+def load_model(model_params, checkpoint_state_dict):
```

Comment:
please use the updated version of master

Reply:
Merged master into the branch today. Says it's up to date.